HHS/OPRE-OPS
# Ops 2367/initial UI

## What changed

Added a document tab to Agreements. The document is currently populated with static data so every agreement has the same list of six documents. This commit contains no major logic for interacting with documents.


## Issue

[OPS-2367](https://app.zenhub.com/workspaces/opre-ops-62d197b4c468970013bcd2ec/issues/gh/hhs/opre-ops/2367)

## How to test

1. Log in as any account
2. Click Agreements in the header
3. Click on any agreement
4. Click on the documents tag

You should see 6 documents in 3 rows of 2. 

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/1977514/37ed465d-b7c0-4220-99c5-38145c32d01f)

## Definition of Done Checklist


## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
